server/src/action/abstract.js

```diff
@@ -7,6 +7,8 @@ class AbstractAction {
       this.validate,
       this._handler.bind(this)
     ]
+
+    this.DEFAULT_ITEM_PER_PAGE = 15
```

Comment:
Would be nice to have this in a configuration file instead

Reply:
it has been moved